<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCaliCurveCover</class>
 <widget class="QWidget" name="QCaliCurveCover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCaliCurveCover</string>
  </property>
  <widget class="QPushButton" name="ok_Button">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>243</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_button">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>243</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Select_Button">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>90</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选中曲线</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="His_Button">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>90</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>历史曲线</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>136</y>
     <width>594</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Whether to overwrite the working curve wigh the current curve?</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
